infra/terraform/global/assets/create_etcd_ebs_snapshot/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"./snapshot"
+	"./tag"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"log"
+	"os"
+)
+
+var (
+	clientSession = session.Must(session.NewSession())
+	connection    = ec2.New(clientSession)
+	optionalTagKey   = os.Getenv("TAG_KEY")
+	optionalTagVal   = os.Getenv("Tag_VALUE")
+)
+
+func main() {
+
+	// Filter instances with role set to master
+	instanceKeyFilter := os.Getenv("INSTANCE_TAG_KEY")
+	instanceValueFilter := os.Getenv("INSTANCE_TAG_VALUE")
```

Comment:
Should these be with the other `var` entries?

Reply:
They are now